File: abc/abc212/a.cpp
// https://atcoder.jp/contests/abc212/submissions/25511522

#include <bits/stdc++.h>

using namespace std;

int main() {
  int A, B;
  cin >> A >> B;
  if(A != 0 && B == 0) {
    cout << "Gold";
  } else if(A == 0 && B != 0) {
    cout << "Silver";
  } else {
    cout << "Alloy";
  }
}

File: abc/abc212/a.rs
// https://atcoder.jp/contests/abc212/submissions/47525207

use proconio::input;

fn main() {
    input! {
        a: u8,
        b: u8
    }

    print!("{}", if 0 < a && b == 0 {"Gold"} else if a == 0 && 0 < b {"Silver"} else {"Alloy"});
}

